Wire-iOS/Sources/UserInterface/Settings/BackupViewController.swift

```diff
@@ -66,7 +70,14 @@ final class BackupStatusCell: UITableViewCell {
 }
 
 final class BackupActionCell: UITableViewCell {
-    let actionTitleLabel = UILabel()
+    let actionTitleLabel: DynamicFontLabel = {
+        let label = DynamicFontLabel(text: "self.settings.history_backup.action".localized,
```

Comment:
For setting the text, it would be a good idea to use:

```suggestion
        let label = DynamicFontLabel(text: L10n.Localizable.Self.Settings.HistoryBackup.Password.description,
```

Reply:
done